### ISSUE TITLE
Have a bug into linter

### ISSUE THREAD

**Issue by KODerFunk** [brigade/haml-lint#76]
Have a bug on fresh project. Ruby 2.2.0 / Rails: 4.2.1
```
undefined method `type' for nil:NilClass
/lib/haml_lint/linter/unnecessary_string_output.rb:34:in `outputs_string_literal?'
```

Stack trace: https://gist.github.com/KODerFunk/66f09d0a2fd63e2565ff

**Comment by sds** [comment]
Hey @KODerFunk, thanks for the report. 

It would help if you included the file that caused the error so we can reproduce this easily on our end. Thanks!

**Comment by KODerFunk** [comment]
The problem disappeared by itself) Thank you for your responsiveness!

**Comment by threadhead** [comment]
+1 Experiencing the same, with the same stack trace.

Happy to include some files, but I'm just running in on a RoR (4.2.1) app: `haml-lint app/**/*.html.haml`

**Comment by threadhead** [comment]
I should add that I am using 0.13.0, which is supposed to fix #71

**Comment by sds** [comment]
Hey @threadhead,

The easiest way to help would be to isolate the file that triggers this problem so I can reproduce it locally, and post the code in this issue. Do that, and I'll happily take a look.

Thanks!
